src/metabase/db/custom_migrations.clj

```diff
@@ -1249,14 +1249,16 @@
 (comment
   ;; How to create `resources/sample-content.edn` used in `CreateSampleContent`
   ;; -----------------------------------------------------------------------------
-  ;; Start a fresh metabase instance without the :ee alias so instance analytics stuff is not created.
+  ;; Check out a fresh metabase instance on the branch of the major version you're targeting,
+  ;; and without the :ee alias so instance analytics content is not created.
   ;; 1. create a collection with dashboards, or import one with (metabase.cmd/import "<path>")
   ;; 2. execute the following to spit out the collection to an EDN file:
   (let [pretty-spit (fn [file-name data]
                       (with-open [writer (io/writer file-name)]
                         (binding [*out* writer]
                           #_{:clj-kondo/ignore [:discouraged-var]}
                           (pprint/pprint data))))
+        columns-to-remove [:view_count]
```

Comment:
`:view_count` was added since the sample content was introduced, and we don't want to export it with the sample content.

Reply:
Some guidance on how to determine whether any future column should be removed could be helpful.

Reply:
seems obvious to me given this example, I'm not sure what I'd write other than this

https://github.com/metabase/metabase/pull/43200/files#diff-be85115f22d8821f0a204e2c870538dc80f6c4191640e6f02c7cc558dbda6544R1290

Reply:
Attempted to make it clearer here [~ add comments](https://github.com/metabase/metabase/pull/43200/commits/c36556ea0d1c1b00983caea36fbd1078514ca18a)